src/utils/getFileTypeFromPath/getFileTypeFromPath.ts

```diff
@@ -1,58 +1,36 @@
-import { FileType } from '../../tree/types';
+import { FileType } from "../../tree/types";
 
 /** Given an inputted path: filters dots, forward slashes, and
  *  analyzes position of them to return the file type of the path
  * @param {string} path the path of a specific file or folder
+ * @param {boolean} isFile boolean to determine if target is folder or a file
  * @returns {FileType} file type such as .FILE, .CONFIG_FILE, or .FOLDER
-*/
+ */
 
-export const getFileTypeFromPath = (path: string): FileType => {
-  // Count amount of periods in the path
-  const numberOfDotsInPath = (path.match(/\./g) || []).length;
-  // Breaks the path up at the forward slashes
-  const forwardSlashPathFilter = path.split('/');
-
-  const configNumberOfDots = 2; // number of dots in a config file path
-  const fileNumberOfDots = 1; // number of dots in a regular file path
-  const folderNumberOfDots = 0; // number of dots in a folder path
+export const getFileTypeFromPath = (
```

Comment:
JSDOC

Reply:
@ralph-dev Isn't the notes after import JSDOC?

Reply:
Add JSDOC for getFileTypefromPath function
/**
* Get's the file type by analyzing the number of "."'s etc
*/